packages/salesforcedx-vscode-lightning/src/index.ts

```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (c) 2017, salesforce.com, inc.
+ * All rights reserved.
+ * Licensed under the BSD 3-Clause license.
+ * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+ */
+
+import * as languageServer from '@salesforce/salesforcedx-slds-linter/out/src/client';
+import * as vscode from 'vscode';
+
+export function activate(context: vscode.ExtensionContext) {
+  console.log('SFDX SLDS Linter Extension Activated');
+  vscode.workspace
+    .findFiles('**/staticresources/*.resource', '**/node_modules/**')
```

Comment:
Why are we looking into the node_modules folder? Do we have control over those?

Reply:
The second arg is actually an exclude param, but looking at it now it doesn't make sense, I'll remove it